modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosControllerEpics.scala

```diff
@@ -350,13 +350,18 @@ object GmosControllerEpics extends GmosEncoders {
           applyParam(state.nsState, encode(config.nsState), DC.setNsState)
         ).flattenOption
 
+      // Don't set CC if Dark or Bias
       private def ccParams(state: GmosCCEpicsState, config: Config[T]#CCConfig): List[IO[Unit]] =
-        (setFilters(state, config.filter) ++
+        if(config.isDarkOrBias) List.empty
```

Comment:
So in a Dark it doesn't really matter if any mechanism is deployed?

Reply:
Yes, because the detector's shutter is closed.